### PR TITLE
Rename MP_* frames to PATH_*

### PR DESCRIPTION
**Pull request by mirjak**
fixes issue #359

### CODE REVIEW

**Review by huitema — APPROVED** [review]
I like it that way, including NEW_PATH_ID.

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -887,34 +887,34 @@ Terrestrial | 100ms  | 350ms
...
+scheduling decisions by the sender of the PATH_ACK frames. In the example
+above, we may assume that the PATH_ACK will be sent over the terrestrial
 link, because that provides the best response time. In that case, the
 computed RTT value for the satellite path will be about 350ms. This
-lower than the 600ms that would be measured if the MP_ACK came over
+lower than the 600ms that would be measured if the PATH_ACK came over
 the satellite channel, but it is still the right value for computing
-for example the PTO timeout: if an MP_ACK is not received after more
-than 350ms, either the data packet or its MP_ACK were probably lost.
+for example the PTO timeout: if an PATH_ACK is not received after more
```
Comment (MikeBishop) [inline draft-ietf-quic-multipath.md:904]:
```suggestion
for example the PTO timeout: if a PATH_ACK is not received after more
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -383,16 +383,16 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
-MP_ACK frames (defined in {{mp-ack-frame}}) can be returned on any path.
-If the MP_ACK is preferred to be sent on the same path as the acknowledged
+PATH_ACK frames (defined in {{mp-ack-frame}}) can be returned on any path.
+If the PATH_ACK is preferred to be sent on the same path as the acknowledged
 packet (see {{compute-rtt}} for further guidance), it can be beneficial
-to bundle an MP_ACK frame with the PATH_RESPONSE frame during
+to bundle an PATH_ACK frame with the PATH_RESPONSE frame during
```
Comment (MikeBishop) [inline draft-ietf-quic-multipath.md:389]:
```suggestion
to bundle a PATH_ACK frame with the PATH_RESPONSE frame during
```

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -1275,14 +1275,14 @@ Path Identifier:
...
+As the PATH_NEW_CONNECTION_ID frames applies the sequence number per path,
+the sequence number in the PATH_RETIRE_CONNECTION_ID frame is also per
+path. The PATH_RETIRE_CONNECTION_ID frame retires the Connection ID with
 the specified Path ID and sequence number.
 
 The processing of an incoming RETIRE_CONNECTION_ID frame
 is described in {{Section 19.17 of QUIC-TRANSPORT}}. The same processing
-applies for MP_RETIRE_CONNECTION_ID frames per path, while the
+applies for PATH_RETIRE_CONNECTION_ID frames per path, while the
 processing of an RETIRE_CONNECTION_ID frame is only applied for Path ID 0.
```
Comment (MikeBishop) [inline draft-ietf-quic-multipath.md:1286]:
```suggestion
processing of a RETIRE_CONNECTION_ID frame is only applied for Path ID 0.
```

**Review by MikeBishop — APPROVED** [review]

File: draft-ietf-quic-multipath.md [hunk 4]
```diff
@@ -196,10 +196,10 @@ to a packet number space.
 
 The same Path ID is used in both directions to
 address a path in the new multipath control frames,
-such as PATH_ABANDON {{path-abandon-frame}}, PATH_BACKUP {{path-backup-frame}}},
-PATH_AVAILABLE {{path-available-frame}} as well as MP_ACK {{mp-ack-frame}}.
-Further, connection IDs are issued per Path ID using the
-MP_NEW_CONNECTION_ID frame (see {{mp-new-conn-id-frame}}).
+such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
+PATH_AVAILABLE {{path-available-frame}} as well as PATH_ACK {{mp-ack-frame}}.
+Further, connection IDs are issued per Path ID ID using the
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:201]:
```suggestion
Further, connection IDs are issued per Path ID using the
```

**Review by Yanmei-Liu — APPROVED** [review]
Looks good!

**Pull request merged**